dataset-variance-resub-2022/edges/dataset.json

```diff
@@ -0,0 +1,61 @@
+{
+  "title": "hiPSC Single-Cell Image Dataset (edge cells)",
+  "version": "2022.1",
+  "name": "variance_resub_edges",
```

Comment:
QUESTION: is this a good `id` for the dataset? The `resub` seems less official

Reply:
I'm not sure if this is documented or explained anywhere but the "name" field is part of the shorthand id name for the data set and it appears in urls as a key for doing any kind of lookup into the data. (e.g. https://cfe.allencell.org/?cellSelectedFor3D=903210&colorBy=cell-line&dataset=aics_hipsc_v2021.1&plotByOnX=cell-volume&plotByOnY=dna-volume where you see `aics_hipsc_v2021.1`.  It's the mostly-human-readable unique ID for the dataset.   The "title" field is the more presentable longer form name for the dataset.

Reply:
Personally I'd like to see something slightly more specific than variance_edges.  Technically any unique name is ok but it WILL be visible in urls and potentially used by external users to also access this data.

Reply:
Do we need to rename the subfolder names if we decide to change this field?

Reply:
No I think the subdirectories are all named in the top level dataset.json and the ids are allowed to be different from the directory names.  (I might be wrong but I will fix that if it turns out to be a problem)